platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/SwirldsPlatform.java

```diff
@@ -366,8 +369,13 @@ public class SwirldsPlatform implements Platform {
         this.selfId = id;
         this.currentAddressBook = initialState.getAddressBook();
 
+        final SignedStateNexus emergencyState = new SignedStateNexus();
+        final Consumer<PlatformStatus> statusChangeConsumer = s -> {
+            notificationEngine.dispatch(PlatformStatusChangeListener.class, new PlatformStatusChangeNotification(s));
+            emergencyState.clear();
```

Comment:
So the state needs to be cleared once so that it use up memory when we don't need it. It only needs to be cleared when:
- we actually have an emergency state
- enough nodes have received the state so that we can reach consensus

Since its simpler and just as cheap to clear it every time, thats what I do.

Reply:
Are we guaranteed to never change platform status at a time when we don't want the emergency state to be cleared? At which status transition do we expect this clear to *actually* be relevant?

Reply:
It was supposed to clear only on ACTIVE, good catch. Fixed.
I also changed the paradigm, since clearing on active is kind of business logic, I made it part of a class that extends the nexus. I will have similar logic in a subsequent PR.